edlib/src/edlib.cpp

```diff
@@ -401,46 +401,45 @@ static inline unsigned char* createReverseCopy(const unsigned char* const seq, c
  * @param [in] Pv  Bitset, Pv[i] == 1 if vin is +1, otherwise Pv[i] == 0.
  * @param [in] Mv  Bitset, Mv[i] == 1 if vin is -1, otherwise Mv[i] == 0.
  * @param [in] Eq  Bitset, Eq[i] == 1 if match, 0 if mismatch.
- * @param [in] hin  Will be +1, 0 or -1.
+ * @param [in] Phin  Bitset, 00..01 when hin == +1.
+ * @param [in] Mhin  Bitset, 00..01 when hin == -1.
  * @param [out] PvOut  Bitset, PvOut[i] == 1 if vout is +1, otherwise PvOut[i] == 0.
  * @param [out] MvOut  Bitset, MvOut[i] == 1 if vout is -1, otherwise MvOut[i] == 0.
- * @param [out] hout  Will be +1, 0 or -1.
+ * @param [out] Phout  Bitset, 00..01 when hout == +1.
+ * @param [out] Mhout  Bitset, 00..01 when hout == -1.
  */
-static inline int calculateBlock(Word Pv, Word Mv, Word Eq, const int hin,
+static inline pair<Word, Word> calculateBlock(Word Pv, Word Mv, Word Eq, const Word Phin, const Word Mhin,
                                  Word &PvOut, Word &MvOut) {
     // hin can be 1, -1 or 0.
     // 1  -> 00...01
     // 0  -> 00...00
     // -1 -> 11...11 (2-complement)
```

Comment:
This is not that meaningful any more, we can maybe rewrite it like this:

```suggestion
    // (Phin, Mhin) can have one of three possible values:
    //   - (1, 0) if hin is 1.
    //   - (0, 1) if hin is -1.
    //   - (0, 0) if hin is 0.
```